qemu/tests/cdrom.py

```diff
@@ -335,6 +335,22 @@ def get_device(vm, dev_file_path):
                 raise error.TestFail("Could not find a valid cdrom device")
         return device
 
+    def get_testing_cdrom_device(session, cdrom_dev_list):
+        """
+        Get the testing cdrom used for eject
+        :param session: VM session
+        :param cdrom_dev_list: cdrom_dev_list
+        """
+        if params["os_type"] == "windows":
+            winutil_drive = utils_misc.get_winutils_vol(session)
+            cdrom_dev_list.remove(str(winutil_drive) + ":")
```

Comment:
make it more readable,  winutils_dirve = "%s:" % winutils_dirve cdrom_dev_list.remove(winutils_drive)